src/main/java/com/google/sps/data/Tag.java

```diff
@@ -2,6 +2,7 @@
 
 public class Tag extends DBObject {
   public static final String HIDDEN_KEY = "hidden";
+  public static final String ID_KEY = "id";
```

Comment:
Could this property be migrated to DBObject, since it's common to all DBObjects?